### PR TITLE
Add `--no-webapps` gulp option

### PR DESCRIPTION
**Pull request by eanders-ms**
Optional flag to skip building authcode, skillmap, multiplayer, and soon kiosk. Usage:
```bash
gulp --no-webapps
```

```bash
gulp watch --no-webapps
```

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -653,6 +653,26 @@ const copyMultiplayerHtml = () => rimraf("webapp/public/multiplayer.html")
 
 const multiplayer = gulp.series(cleanMultiplayer, buildMultiplayer, gulp.series(copyMultiplayerCss, copyMultiplayerJs, copyMultiplayerHtml));
 
+/********************************************************
+                 Webapp build wrappers
+*********************************************************/
+
+const shouldBuildWebapps = () => (process.argv.indexOf("--no-webapps") === -1);
```
Comment (jwunderl) [inline gulpfile.js:660]: Nit; making this an async function would remove the need for an explicit returned noop / inverted logic, but fine either way

**Review by jwunderl — APPROVED** [review]
Nice

**Review by kimprice — APPROVED** [review]
This is awesome!

File: gulpfile.js (hunk 1 above)
Comment (eanders-ms) [reply gulpfile.js:660]: Something we could improve later for sure. Gulp's expectations here are very specific and I wrestled with it more than I'd expected to.

**Review by thsparks — APPROVED** [review]

**Pull request merged**